doc/nrf/libraries/networking/nrf_cloud_alerts.rst

```diff
@@ -69,11 +69,11 @@ To use this library, complete the following steps:
 #. Define a local variable of type :c:struct:`nrf_cloud_alert_info`.
 #. Initialize the structure members:
 
-  * :c:member:`type` - A value of the :c:enum:`nrf_cloud_alert_type` enum.
-  * :c:member:`value` - Use 0 if not otherwise needed.
-  * :c:member:`description` -Use NULL to suppress transmission of this field.
+   * :c:member:`type` - A value of the :c:enum:`nrf_cloud_alert_type` enum.
+   * :c:member:`value` - Use 0 if not otherwise needed.
```

Comment:
```suggestion
   * :c:member:`value` - Use ``0`` if not otherwise needed.
```
Since these are values should we mark it (?)

Reply:
Good catch. We definitely need the backticks.